src/ml/neural_net/style_transfer/mps_style_transfer_encoding_node.m

```diff
@@ -0,0 +1,49 @@
+#include <ml/neural_net/style_transfer/mps_style_transfer_encoding_node.h>
+#include <ml/neural_net/mps_layer_helper.h>
+
+@implementation TCMPSStyleTransferEncodingNode
+
+- (instancetype) initWithParameters:(NSString *)name
+                          inputNode:(MPSNNImageNode *)inputNode
+                             device:(id<MTLDevice>)dev
+                          cmd_queue:(id<MTLCommandQueue>)cmd_q
+                         descriptor:(TCMPSEncodingDescriptor *)descriptor
+                        initWeights:(NSDictionary<NSString *, NSData *> *) weights {
+  @autoreleasepool {
+    
+    self = [super init];
+
+    MPSCNNConvolutionNode *conv = [MPSCNNConvolutionNode createConvolutional:inputNode
+                                                            kernelWidth:descriptor.conv.kernelWidth
+                                                           kernelHeight:descriptor.conv.kernelHeight
+                                                   inputFeatureChannels:descriptor.conv.inputFeatureChannels
+                                                  outputFeatureChannels:descriptor.conv.outputFeatureChannels
+                                                            strideWidth:descriptor.conv.strideWidth
+                                                           strideHeight:descriptor.conv.strideHeight
+                                                           paddingWidth:descriptor.conv.paddingWidth
+                                                          paddingHeight:descriptor.conv.paddingHeight
+                                                                weights:(float *)weights[@"encoding_conv_weights"].bytes
```

Comment:
Hmm, should this argument take `NSData` after all?

Reply:
I think float pointer. I'll change it!

Reply:
I've been looking into it and memory management stuff is weird here. Keeping it as NSData might help us out with the ARC stuff (even in the calling code).